start.py

```diff
@@ -89,6 +91,15 @@ def main(args):
         for node in nodes:
             node.volumes.append(java_image)
 
+    if args.spark2_version:
+        logger.info('args.spark2_version = %s', args.spark2_version)
```

Comment:
Not sure this is needed since we already log args in DEBUG mode.

Reply:
Removed the line.